app/controllers/concerns/vendor_api/application_data_concerns.rb

```diff
@@ -0,0 +1,45 @@
+module VendorAPI
+  module ApplicationDataConcerns
+  private
+
+    def application_choice_id
+      params.permit(:application_id)[:application_id]
+    end
+
+    def application_choice
+      @application_choice ||= application_choices_visible_to_provider.find(application_choice_id)
+    end
+
+    def render_application
+      render json: application_json
+    end
+
+    def application_json
+      data = { data: ApplicationPresenter.new(version_number, application_choice).as_json }
```

Comment:
what are the repercussions of removing `serialized_json` and caching? cc @steventux

Reply:
this does come from the cache, it's just not yet converted to json. The method name is confusion and we should probably rename it! `as_json` -> `as_hash`

https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/presenters/vendor_api/application_presenter.rb#L28

Reply:
There's less of a performance hit rendering JSON for a single application, the index was the place where we found a performance gain by caching a string for each application. Of course we won't get the benefit of fetching single applications from the cache which was previously populated by the index action. Vendors seem to be using the /applications endpoint more than the single application endpoint so I think it's fine.